source/whats-new.txt

```diff
@@ -24,8 +24,13 @@ What's New in 4.3
 
 New features of the 4.3 Java driver release include:
 
-- MongoDB Versioned API. For more information, see our :doc:`Versioned API guide </fundamentals/versioned-api>`. 
-- Added a builder API for the ``setWindowFields`` pipeline stage to allow the use of window operators 
+- Added support for MongoDB Versioned API. For more information, see our
+  :doc:`Versioned API guide </fundamentals/versioned-api>`.
+- Added connection support for
```

Comment:
I find `Added connection support` to be confusing.   Change to either `Added support` or `Added support for connecting to`?